spec/fixtures/vcr_cassettes/SolidusBolt_Oauth_TokenService/_call/makes_the_API_call.yml

```diff
@@ -0,0 +1,59 @@
+---
+http_interactions:
+- request:
+    method: post
+    uri: https://api-sandbox.bolt.com/v1/oauth/token
+    body:
+      encoding: UTF-8
+      string: grant_type=authorization_code&code=Bolt%20Authorization%20Code&scope=openid%20bolt.account.manage&client_id=CB_FtfGdUP2e.UwZWGwaPAuFB.923d99a0f8bef8834390bd7423ab03efa8050c86f2dc60cb664d48fcca28a7b5&client_secret=db1bd1ff3631f9c3e30d3555ad2989fa7cbed58b2344aea911aa7e2d52dba3e3
```

Comment:
Can you obfuscate the Client_id and client_secret?

Reply:
done

Reply:
Got it
Modified it now